src/main/java/sg/ncl/MainController.java

```diff
@@ -4559,7 +4617,7 @@ private boolean isMemberJoinRequestPending(String loginUserId, String json) {
                 return true;
             }
         }
-        log.info("User: {} is viewing experiment page", loginUserId);
+        //log.info("User: {} is viewing experiment page", loginUserId);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)